### PR TITLE
Skip video partition on redump-style images

### PR DESCRIPTION
**Pull request by 18woldemar**

### CODE REVIEW

**Comment by InternalLoss** [comment]
I can't comment on the code style itself but I can confirm this works flawlessly on macOS (Thank you so much!)

**Comment by GXTX** [comment]
Split this into 2 PRs please. One for the offset change and the other for adding Android support (although I don't know if we should bother with that).

File: extract-xiso.c [hunk 1]
```diff
@@ -58,176 +58,6 @@
 	 * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 	 * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 	 *\
-
```
Comment (GXTX) [inline extract-xiso.c:61]: Leave this as it's part of the copyright header.

File: extract-xiso.c [hunk 2]
```diff
@@ -373,7 +221,7 @@
 #endif
 
 
-#define exiso_version					"2.7.1 (01.11.14)"
+#define exiso_version					"2.7.2 (02.04.21)"
```
Comment (GXTX) [inline extract-xiso.c:224]: Don't change this.

**Comment by dracc** [comment]
Changes look good, but I'd prefer to see them split into separate pull requests as they seem to be quite different in nature.

The splits I'm thinking would be reasonable;
Removing changelog
Adding Android support
Adding redump style support

**Comment by LukeUsher** [comment]
Closing this as it is (for the most part) taken care of by #47 and #41

**Pull request closed without merging**